cmd/devtool/devtool.go

```diff
@@ -159,7 +160,16 @@ func ethSetup(ethAcctAddr, keystoreDir string, isBroadcaster bool) {
 	}
 	glog.Infof("Using controller address %s", ethController)
 
-	client, err := eth.NewClient(ethcommon.HexToAddress(ethAcctAddr), keystoreDir, passphrase, backend,
+	gpm := eth.NewGasPriceMonitor(backend, 5*time.Second, big.NewInt(0))
+	// Start gas price monitor
+	_, err = gpm.Start(context.TODO())
```

Comment:
Is context.TODO() here actually meant to be a placeholder or would it be fine to just use context.Background()?

I believe both are technically equivalent, but typically TODO() is used to serve as a placeholder with the expectation that it will be replaced while Background() is used when an empty context is explicitly desired.

Reply:
fixed in 0b64e17